docs/changelog/1664.md

```diff
@@ -1 +1 @@
-- Added communication of substeps data for waveform coupling. A new attribute `substeps` in the `exchange` tag allows deactivating the additional communication.
+- Added communication of substeps data for waveform coupling. A new attribute `substeps` in the `exchange` tag allows activating and deactivating this additional communication. `substeps` defaults to `false`, since not all acceleration schemes support this feature. See https://github.com/precice/precice/issues/1825 for details.
```

Comment:
```suggestion
- Added communication of substeps data for waveform coupling. A new attribute `substeps` in the `exchange` tag allows activating and deactivating this additional communication. `substeps` defaults to `false`, since not yet all acceleration schemes support this feature. See https://github.com/precice/precice/issues/1825 for details.
```